core/templates/dev/head/pages/profile/profile.html

```diff
@@ -100,9 +100,8 @@ <h2 class="oppia-profile-username-small-screen">
       <hr class="oppia-profile-stat-container-line-small-screen">
     </md-card>
 
-    <md-card class="oppia-profile-content-card">
-      <strong>Portfolio</strong>
-      <md-content class="oppia-profile-portfolio-container">
+    <md-card class="oppia-profile-content-card" style="margin-top: 0; padding-top: 0;">
```

Comment:
Just to check: can either of these inline styles go in the CSS file, or would they get overridden? (If the latter, perhaps you could perhaps make the CSS rule more specific, e.g. `md-card.oppia-profile-content-card`?)